pom.xml

```diff
@@ -122,9 +122,10 @@
         <guava.version>18.0</guava.version>
         <findbugs.version>3.0.1</findbugs.version>
         <commons-codec.version>1.10</commons-codec.version>
-        <spring.version>4.3.5.RELEASE</spring.version>
-        <jetty.version>9.3.14.v20161028</jetty.version>
+        <spring.version>4.3.12.RELEASE</spring.version>
+        <jetty.version>9.3.21.v20170918</jetty.version>
```

Comment:
❓ Can we make sure the UI compression still works. Can't remember for sure but I think we had to keep that old Jetty version to make sure http compression still worked for the UI. Might have been only in specific pom files though (https://github.com/codice/ddf/pull/2159).

Reply:
Verified via the steps in #2159 that gzip is still working.